cl/citations/management/commands/import_citations_csv.py

```diff
@@ -78,7 +84,18 @@ def process_csv_data(
             continue
 
         if cluster_id and citation_to_add:
-            add_citations_to_cluster([citation_to_add], cluster_id)
+            try:
+                add_citations_to_cluster([citation_to_add], cluster_id)
+            except Exception as e:
+                if "Field 'volume' expected" in str(e):
+                    # Fail silently, we already know this issue
+                    logger.info(
+                        f"Row: {index} - Invalid volume in citation: {citation_to_add} for cluster: {cluster_id}"
+                    )
```

Comment:
I think we try to avoid any naked exception.  What is the exception that is thrown here?

Reply:
I want to catch any possible exceptions (in order to identify possible errors), but only handle the exception that is raised when we get a volume number with dashes (not supported by cl), it throws a ValueError but i don't want to be that specific in case we raise that exception by a different reason.